Centaurus.Test.Domain/AlphaMessageHandlersTests.cs

```diff
@@ -15,6 +15,7 @@ public class AlphaMessageHandlersTests : BaseMessageHandlerTests
         [SetUp]
         public void Setup()
         {
+            EnvironmentHelper.SetTestEnvironmentVariable();
```

Comment:
Instead of setting environment variables in each test fixture, define a one-time `SetUpFixture` for the `Centaurus.Test` namespace. It will run once before all tests defined in this namespace, so we just need to check that all tests are defined under `Centaurus.Test` to make it work automagically. Docs ref: https://github.com/nunit/docs/wiki/SetUpFixture-Attribute

```c#
using System;
using NUnit.Framework;

namespace Centaurus.Test
{
    [SetUpFixture]
    public class MySetUpClass
    {
        [OneTimeSetUp]
        public void SetupTestEnv()
        {
	    EnvironmentHelper.SetTestEnvironmentVariable();
        }
    }
}
```